domain/resourcegroup/runaway.go

```diff
@@ -183,9 +402,18 @@ func (rm *RunawayManager) QuarantineRecordChan() <-chan *QuarantineRecord {
 	return rm.quarantineChan
 }
 
+// StaleQuarantineRecordChan returns the channel of staleQuarantineRecord
+func (rm *RunawayManager) StaleQuarantineRecordChan() <-chan *QuarantineRecord {
+	return rm.staleQuarantineRecord
+}
+
 // examineWatchList check whether the query is in watch list.
-func (rm *RunawayManager) examineWatchList(resourceGroupName string, convict string) bool {
-	return rm.watchList.Get(resourceGroupName+"/"+convict) != nil
+func (rm *RunawayManager) examineWatchList(resourceGroupName string, convict string) (bool, rmpb.RunawayAction) {
+	item := rm.getWatchFromWatchList(resourceGroupName + "/" + convict)
+	if item == nil {
+		return false, 0
+	}
+	return true, item.Action
```

Comment:
For a manual add item without action, the action is none, in this case you should use the setting's action instead. BTW, if we support let manual record reflect the change of query limit's action change, I would expect watch record reflect this change too.

Reply:
You can see L461-463. 
make sense, I will update it.

Reply:
done